buildSrc/build.gradle.kts

```diff
@@ -48,7 +34,7 @@ dependencies {
   implementation(libs.pluginPackages.checkerFramework)
   implementation(libs.pluginPackages.grgit)
   implementation(libs.pluginPackages.jetbrains.changelog)
-  implementation(libs.pluginPackages.jetbrains.intellij)
+  // implementation(libs.pluginPackages.jetbrains.intellij)
```

Comment:
TBD if this can be saved somehow (maybe use `ext` block for reusable IntelliJ configuration? if buildSrc/ isn't feasible)

Reply:
Precompiled Script Plugins might be a solution (?)

Reply:
But I'd rather get it to work in buildSrc/ as with all other multi-project configuration

Reply:
Or simply apply the IntelliJ plugin everywhere, via `allprojects { ... }`

Reply:
Ended up doing the latter